crates/store/re_protos/proto/rerun/v0/catalog.proto

```diff
@@ -0,0 +1,249 @@
+syntax = "proto3";
+
+package rerun.redap.v0;
+
+import "rerun/v0/common.proto";
+
+// The RerunCatalog service is a common API set exposed by:
+// - The Rerun Catalog Server
+// - Any Rerun Storage Node
+//
+// Some Important terminology:
```

Comment:
Maybe add the definition of _Property_ (which I think is what we used to call  _Metadata_, right?).